.sage/main.go

```diff
@@ -51,6 +52,22 @@ func GoLint(ctx context.Context) error {
 	return sggolangcilint.Run(ctx)
 }
 
+func GoFix(ctx context.Context) error {
+	sg.Logger(ctx).Println("fixing Go files...")
+	return sggolangcilint.Fix(ctx)
+}
+
+func GoLines(ctx context.Context) error {
+	sg.Logger(ctx).Println("wrapping Go lines...")
+	return sggolines.Run(ctx)
+}
+
```

Comment:
Do we need these 2? If the goal is to have a GoFormat, maybe inline these 2?

Also `go fix` is an [actual command](https://pkg.go.dev/cmd/fix) in the Go cli so if we keep it, we should avoid some confusion and call it something different.

Reply:
I'm thinking it _might_be useful to have easy access to run them separately in the case of formatting conflicts, but I also think it is `GoFormat` we are primarily looking for, so inlining sounds good to me.

Reply:
Sure we can inline

Reply:
Great. My thinking here is that we will rarely, if ever, want to run one or the other separately so let's start by inlining and avoid bloating the default setup even more if the primary target is to run both together

Reply:
💯 agree

Reply:
I just realized something. Combining these two are not necessary "formatting". For example, we recently added the [`protogetter`](https://github.com/ghostiam/protogetter) linter which has auto-fix capabilities.

Maybe `GoFormat` or `make go-format` is a little bit misleading, as I understand that protogetter changes the behavior, not just formatting.

Reply:
I think its good to have them separately, so the user can choose, but also supply one target to run all formatters, but if you dont agree you can inline.

Reply:
> The default config that gets generated is meant to be a batteries included setup for the most basic Go repos so I think it makes more sense here to inline following Go convention of 1 way of doing things.

To avoid confusion, maybe let's not call it `GoFormat` then, as not only formatting is (and likely will continue to be) applied when executing it?

Reply:
Any suggestions on a better name? Could potentially be called `LinterFix`?

Reply:
I think let's go with `LinterFix` or `LintFix` (only because it's shorter 😄), what do you think @odsod ?

By not prefixing with `Go`, does it imply that this could in the future be extended with e.g. Python linting/formatting?

Reply:
We already have `go-mod-tidy`, `go-test`, `go-lint`, `go-review`, `go-licenses` and more. Why not `go-fix`?

Reply:
Since we are talking about inlining anyway, the only target we'll add here is `go-format` so this discussion is moot?

Reply:
> this discussion is moot?

Well, the protogetter linter will change _behavior_, not formatting. So I still don't think calling this `go-format` is correct.

Reply:
I think I'll leave the naming up to you to decide 😄